generators/client/templates/react/src/main/webapp/app/modules/account/sessions/sessions.tsx.ejs

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Alert, Table, Button, Row } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+import { getSession } from '../../../shared/reducers/authentication';
+import { findAll, invalidateSession } from './sessions.reducer';
+
+export interface ISessionsProps {
+  getSession: Function;
+  findAll: Function;
+  invalidateSession: Function;
+  account: any;
+  sessions: any;
+  updateSuccess: boolean;
+  updateFailure: boolean;
+}
+
+export class SessionsPage extends React.Component<ISessionsProps> {
+  componentDidMount() {
+    this.props.getSession();
+    this.props.findAll();
+  }
+
+  doSessionInvalidation = series => {
+    this.props.invalidateSession(series);
+    this.props.findAll();
+  };
+
+  refreshList = () => {
+    this.props.findAll();
+  };
+
+  render() {
+    const { account, sessions, updateSuccess, updateFailure } = this.props;
+    return (
+      <div>
+        <h2>Active sessions for [<b>{account.login}</b>]</h2>
```

Comment:
doesnt this have translation?

Reply:
Yes, it has but afaik there's no way to put variable in translation in React. I could put a replace if you want but that's a kinda ugly way to deal with this.